databroker/src/grpc/kuksa_val_v2/conversions.rs

```diff
@@ -45,7 +45,10 @@ impl From<&proto::Datapoint> for broker::Datapoint {
 impl From<broker::Datapoint> for Option<proto::Datapoint> {
     fn from(from: broker::Datapoint) -> Self {
         match from.value {
-            broker::DataValue::NotAvailable => None,
+            broker::DataValue::NotAvailable => Some(proto::Datapoint {
+                value: None,
+                timestamp: None,
```

Comment:
why should it not have a timestamp?

Reply:
Fo "normal" DataPoints that has a value the timestamp represents the time the databroker registered that value (doesn't it?). In this case Databroker "deep level magic" has no value and provide no timestamp (I think). We could opt for adding current time if we like to show that it was None at Now, but "Now" is the only time we have.

But a good point to dicuss, do we want a timestamp when return Value==None and if so what Timestamp. (Theoretically we could request Databroker to keep track of when a Value change from something to None, but I do not think it do it at the moment)

Reply:
It actually seems to return something, at least in some cases, so I will see if I can use it

Reply:
So it returns registration time for the signal for "virgin" signals, we could use that.

Reply:
yes, that's what I thought of and IF the signal provider would go away you would have a timestamp as well.

Reply:
> So it returns registration time for the signal for "virgin" signals, we could use that.

Setting the value to `NotAvailable` seems to behave just like every other value, at least when it comes to `sdv.databroker.v1` (and hence it's supported by the core of databroker).

* Setting a value to `20.0` will record the timestamp om when that happened.
* Setting a value to `NotAvailable` will record the timestamp of when that happened.